firefox-ios/Client/Info.plist

```diff
@@ -55,6 +55,9 @@
 	<false/>
 	<key>LSApplicationQueriesSchemes</key>
 	<array>
+		<string>paypal</string>
+		<string>instagram</string>
+		<string>youtube</string>
```

Comment:
I'm a bit confused: this might work for these specific URL schemes but there is a wide variety of schemes for other apps, wouldn't they also suffer from a similar problem?

Reply:
Yes, you right but this is the way that we can find if an app is installed or not. This apps should be manually added in this list and yea... may be other apps too.